python/foxglove-sdk/src/websocket.rs

```diff
@@ -809,9 +809,9 @@ impl PyMessageSchema {
 pub enum PyParameterType {
     /// A byte array.
     ByteArray,
-    /// A decimal or integer value that can be represented as a `float64`.
+    /// A floating-point value that can be represented as a `float64`.
```

Comment:
I'm probably missing some context, but I don't understand why we have a new integer type in `ParameterValue`, but not in `ParameterType`. Maybe right now we're assuming the client originates these values, and we handle the conversion, but it seems like I can't directly construct an integer `Parameter`, even though the interface suggests that I should be able to.

If that's an explicit choice, it's fine, but I think we should document it. And this docstring should probably remain the same.

```py
```